### PR TITLE
Fix English tutorial documentation

### PR DESCRIPTION
**Pull request by navn-r**
Fixes a typo and a broken image link

### CODE REVIEW

File: src/main/resources/doc/en/html/guide/tutorial/tutor-gates.html [hunk 1]
```diff
@@ -46,7 +46,7 @@ <h2>
         I left a little space between the NOT gates and the AND gates; if you want to, though, you can put them up against each other and save yourself the effort of connecting them with a wire later.
       </p>
       <p>
-        Now we want to add the two inputs <var>x</var> and <var>y</var> into the diagram. Select the <b class=propertie>"Input"</b> tool (<img class="icontxt" src="../../../../icons/1616/toolpininput.png" alt="#########">), and place the pins down. You should also place an output pin next to the OR gate's output using the <b class=propertie>"Output"</b> tool (<img class="icontxt" src=
+        Now we want to add the two inputs <var>x</var> and <var>y</var> into the diagram. Select the <b class=propertie>"Input"</b> tool (<img class="icontxt" src="../../../../icons/1616/toolpinintput.png" alt="#########">), and place the pins down. You should also place an output pin next to the OR gate's output using the <b class=propertie>"Output"</b> tool (<img class="icontxt" src=
```
Comment (MarcinOrlowski) [inline src/main/resources/doc/en/html/guide/tutorial/tutor-gates.html:49]: I do not think this is correct way to fix that. I'd rather rename the image (and fix other references to badly spelled file name).

**Review by MarcinOrlowski — CHANGES_REQUESTED** [review]

File: src/main/resources/doc/en/html/guide/tutorial/tutor-gates.html (hunk 1 above)
Comment (navn-r) [reply src/main/resources/doc/en/html/guide/tutorial/tutor-gates.html]: Fixed in 240601c1fd493aacb83ce41a22c898cd4f0d9bb8

**Review by MarcinOrlowski — CHANGES_REQUESTED** [review]
What about the references to renamed PNG images?

**Comment by MarcinOrlowski** [comment]
@navn-r Ping. Do you have plans to update this PR?

**Comment by maehne** [comment]
@navn-r: Anyway, thanks for pointing us to these typos! We pick up this PR and will add remaining fixes. So, please don't close it or delete your branch containing these fixes.

**Pull request merged**